src/Utilities/TMPL.hpp

```diff
@@ -526,6 +526,37 @@ using flat_any = std::integral_constant<
  */
 template <bool... Bs>
 constexpr bool flat_any_v = flat_any<Bs...>::value;
+
```

Comment:
we'll use `tmpl::for_each` and put in a PR with brigand to enable passing in args there.